A nasty workaround for the faulty test.

Related to https://www.postgresql.org/message-id/20220304.113347.2105652521035137491.horikyota.ntt@gmail.com
but pg_terminate_backend() accepts the second argument since PostgreSQL 14.
--- a/src/test/isolation/expected/temp-schema-cleanup.out
+++ b/src/test/isolation/expected/temp-schema-cleanup.out
@@ -86,6 +86,7 @@
     SELECT pg_terminate_backend(pg_backend_pid());
 
 FATAL:  terminating connection due to administrator command
+FATAL:  terminating connection due to administrator command
 server closed the connection unexpectedly
 	This probably means the server terminated abnormally
 	before or while processing the request.
